acoustid/data/track.py

```
@@ -271,8 +273,10 @@
         columns.append(
             sql.func.every(schema.track_mbid.c.disabled).label("all_disabled")
         )
-    query = sql.select(
-        columns, tab.c.track_id.in_(source_ids + [target_id]), group_by=col
+    query = (
+        sql.select(*columns)
+        .where(tab.c.track_id.in_(source_ids + [target_id]))
```

Ruff (0.8.2)

Consider `[*source_ids, target_id]` instead of concatenation

Replace with `[*source_ids, target_id]`

(RUF005)